commit 51928da5f0457a8a593d7ba7f947fb96e97fd06f
Author: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date:   Wed Jul 22 10:19:15 2020 +0200

    common: fix LIBFABRIC flags
    
    If BUILD_RPMEM is set, LIBFABRIC_* are not set up by common.inc and cause rpmemd to fail building
    
    Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>

diff --git src/common.inc src/common.inc
index 6c0d174904b1..8112512aed06 100644
--- src/common.inc
+++ src/common.inc
@@ -311,6 +311,7 @@ LIBFABRIC_MIN_VERSION := 1.4.2
 # utils/docker/images/install-libfabric.sh.
 ifeq ($(BUILD_RPMEM),)
 BUILD_RPMEM := $(call check_package, libfabric --atleast-version=$(LIBFABRIC_MIN_VERSION))
+endif
 ifneq ($(BUILD_RPMEM),y)
 export BUILD_RPMEM_INFO := libfabric (version >= $(LIBFABRIC_MIN_VERSION)) is missing -- \
 see src/librpmem/README for details
@@ -320,7 +321,6 @@ LIBFABRIC_LD_LIBRARY_PATHS := $(shell $(PKG_CONFIG) --variable=libdir libfabric)
 LIBFABRIC_LIBS := $(shell $(PKG_CONFIG) --libs libfabric)
 LIBFABRIC_PATH := $(shell $(PKG_CONFIG) --variable=exec_prefix libfabric)/bin
 endif
-endif
 export BUILD_RPMEM
 export LIBFABRIC_CFLAGS
 export LIBFABRIC_LD_LIBRARY_PATHS
